src/main/webapp/app/tasks/task-list/task-list.component.html

```diff
@@ -1,9 +1,47 @@
 <mat-list data-cy="task-list">
-  <mat-list-item *ngFor="let task of tasks" class="mat-elevation-z1">
-    <mat-icon mat-list-icon>assignment</mat-icon>
-    <h4 mat-line>{{task.name}}</h4>
-    <button mat-icon-button aria-label="Delete task" color="primary">
-      <mat-icon aria-label="Delete task" (click)="delete(task)">delete</mat-icon>
-    </button>
-  </mat-list-item>
-</mat-list>
+  <section *ngIf="pendingTasks?.length>0" class="completed-lists">
```

Comment:
what do you think of just using *ngIf="pendingTasks" and not checking the length?

Reply:
We return an empty array if there is no value in local storage. That's why there is an extra check of length